docs/share.md

```diff
@@ -12,19 +12,28 @@ You can share the data your connector accesses by connecting to the data, creati
 
 Or, if you prefer to have your users test the connector, you can place the connector files in the directories and then run the commands as follows:
 
-**Tableau Desktop:** This is the same as developing the connector.
-Place the files into a subdirectory you create, and then use the -DConnectPluginsPath command line argument.  For example, place your plugin files in `C:\tableau_connectors\myconnector` and then start Tableau:
+**Tableau Desktop:** 
+
+1. Place the files into a subdirectory you create.
```

Comment:
I think we're losing some information here. The line "For example, place your plugin files in `C:\tableau_connectors\myconnector`" is removed, and now it's a bit more vague. The problem here was some confusion over where the command line argument should point to. My suggestion (wording may not be very good):

1. Create a tableau connector directory (For example, C:\tableau_connectors)
2. Place the folder containing your connector's manifest.xml file in this directory. Each connector should have its own folder. (For example, C:\tableau_connectors\my_connector)
3. Run tableau using the -DConnectPluginsPath command line argument, pointing to your connector directory. (For example, -DConnectPluginsPath=C:\tableau_connectors

Reply:
Great suggestions, thanks!

Reply:
+1 Incorporated changes.